dev-qt/qtwebengine/qtwebengine-5.9.1.ebuild

```diff
@@ -74,6 +74,9 @@ DEPEND="${RDEPEND}
 src_prepare() {
 	use pax_kernel && PATCHES+=( "${FILESDIR}/${PN}-5.9.0-paxmark-mksnapshot.patch" )
 
+	# bug 620444 - ensure local headers are used
+	find "${S}" -name "*.pr*" | xargs sed -i -e 's|INCLUDEPATH += |&$$QTWEBENGINE_ROOT/include |' src/webengine/webengine.pro || die
```

Comment:
what's the last `src/webengine/webengine.pro`? isn't it matched by the find?

I think you can use `*.pr[fio]` instead of `*.pr*`, and add `-type f`, but it's not a big deal.